### PR TITLE
Add .footer module

### PR DESCRIPTION
**Pull request by esarajyan**
This PR adds a module for the footer. 

This is a draft of the footer, so it is not responsive to scale changes. 

**Preview**

image

### CODE REVIEW

**Comment by Tara-Tiong** [comment]
asdfsadfsda
# Suggestions :
>you may have already thought about these, these are mostly just tweaks to the current design
- set a minimum width (`min-width: ___px;`) to `.footer` --> will likely enable you to scale it (you probably know that already lol)
- maybe to keep with consistency, you should use something like (but not limited to) `footer--centerColumn`,`footer--centerDiv` (the possibilities are endless) instead of `.footer--div-center`
- change the font colors under the BAMP logo to match Figma --> `#303030`
- add `<br>` in between the address commas/lines to match Figma more 
- you can set the margin of the address paragraph to 0, this will remove the space between **Headquarters** and "_3463 San Pablo Ave. Oakland, CA 94608 (510) 965-5315_" 

### General Comment/Question
(unprofessional comment-->) Looks really cool :DDD
- just curious, why is the `width` on line 15 (CSS) oddly specific?

**Review by johncip — CHANGES_REQUESTED** [review]
hey, here are my comments:

(some of these duplicate Tara's)

**small stuff**

- [x] less vertical space after the word "headquarters"
- [x] we need a little spacing between the button links in the center section
- [x] the text should be our ink color (#303030)
- [x] there should be a line breaks in the address
- [x] you can remove the `<div class="footer-center">`
- [x] let's call the right section `--rightSection` or something similar
- [x] you can put all of the properties for the "normal" links in the class (i.e. move `display: block` & `width: 50%` in there)
- [x] I would call the "normal" links "nav" links etc. to be more specific


**resize behavior & layout**

here's what we should shoot for:

- when resizing the browser window, the email form should not shrink as soon (a good strategy might be to shrink the center sooner)
- each footer section should be centered within its whitespace (not necessarily text centering)

**Comment by johncip** [comment]
FYI, I resolved a merge conflict that looked like this:

```css
.btnLink-red {
<<<<<<< footer
    background: var(--red) !important; 
=======
    background: var(--red) !important;
>>>>>>> master
}
```

I think what happened is:

1. you made the same `!important` fix to `.btnLink-red` in this branch (in commit ecc6dc8b8f3398635d4d1e7f464bc2fbc63c98e1)
2. one of the versions had trailing whitespace and the other didn't.

I just want to make you aware of a few good strategies for dealing with changes in other branches that you want to incorporate (duplicating is ok, but the above is an example of what can go wrong). Besides duplicating, in order of simplicity:

1. wait until the fix lands in master, and then merge master (simplest, but you have to wait)
2. merge the other feature branch (in this case `btnLink-fix`) into your feature branch, on the assumption that it will land in master. (ok if the other branch is small)
3. **base** your feature branch on the other feature branch. this only works if you haven't made it yet, or are able to rebase (`git rebase`). better if the other branch is large.

In the 2nd and 3rd cases, you'd probably mark the PR as "don't merge" and point out somewhere that it depends on another PR. Depending on how strict the people on that project are about PRs only dealing with a single issue.

File: style/modules/footer.css
Comment (johncip) [inline style/modules/footer.css]: `nowrap` is the default, so it can come out (also the one under `.footer--midSection`)

File: style/modules/footer.css
Comment (johncip) [inline style/modules/footer.css]: this can be `var(--light-gray)` now that the variables PR is in

**Comment by johncip** [comment]
Looks good! As discussed, let's treat further polishing up the collapse behavior as a separate task.

**Review by johncip — APPROVED** [review]

**Pull request merged**
